timeless/templates/auth/login.html

```diff
@@ -1,20 +1,23 @@
-{% extends 'base.html' %}
-
-{# @todo #283:30min Change the style of login page. Lets change the#}
-{#  style of a login page to get the UX like demonstrated in a picture below#}
-{#  https://drive.google.com/uc?export=view&id=1oh8cLv-vHTCBmb0RDkHQiyMythC4_M6o#}
-{% cache 600 %}
-{% block header %}
-  <h1>{% block title %}Log In{% endblock %}</h1>
-{% endblock %}
-
-{% block content %}
+<!doctype html>
+<title>{% block title %}{% endblock %} - Timlessis</title>
+<link rel="stylesheet" href="{{ url_for('static', filename='login.css') }}">
+<!--
+@todo #460:30min Change the style of login page. Make the image for the
+ login page like shown in UX documentation, then apply the image to the
+ background of login page so it becomes equals to the image below
+ https://drive.google.com/uc?export=view&id=1oh8cLv-vHTCBmb0RDkHQiyMythC4_M6o
```

Comment:
@paulodamaso wording in this puzzle is quite strange and has a few grammar mistakes. How about: "Change the style of the login page. Create the image specified in UX documentation, then apply oit as background. UX mock: [google docs link"